lib/resources/file.rb

```diff
@@ -20,11 +33,13 @@ class FileResource < Inspec.resource(1) # rubocop:disable Metrics/ClassLength
         its('mode') { should cmp '0644' }
       end
     "
-    include MountParser
 
     attr_reader :file, :mount_options
     def initialize(path)
+      # select permissions style
+      @perms_provider = select_file_perms_style(inspec.os)
       @file = inspec.backend.file(path)
+      return skip_resource 'The `file` resource is not supported on your OS yet.' if @perms_provider.nil?
```

Comment:
I think we should not skip the resource if the no perm_provider is available. Most of the file implementation is done in train anyway. I would prefer to move this skip return to the individual functions that cannot be executed aka `readable?` etc